docs/source/models/cause.rst

```diff
@@ -171,29 +174,321 @@ Common data sources for cause models
 
 .. todo::
 
-   Format as table with measure, measure definition, data sources and
-   their uses.
+   Update mortality-related data sources within existing format.
+
+Once a cause model structure is specified, data is needed to inform its states
+and transitions. For our purposes, cause models generally have the following
+data needs:
+
+#. The probability that a simulant will start the simulation in a given state
+   within the cause model.
+#. The probability that a simulant will transition to a new state within the
+   cause model in a given time-step.
+#. The disability weight for each state in the cause model
+#. The probability that a simulant in a given cause model state will die in a
+   given time-step.
+
+There are several common data sources that can be used for these needs, which
+are outlined in the table below and discussed in more detail afterward.
+`Non-standard data sources`_ are discussed later on this page.
+
+.. list-table:: Data Definitions
+   :widths: 20 30 30
+   :header-rows: 1
+
+   * - Measure
+     - Definition
+     - Uses
+   * - `Prevalence`_
+     - Proportion of population with a given condition.
+     - Initialize cause model states.
+   * - `Birth Prevalence`_
+     - Proportion of all live births born with a given condition.
+     - Initialize neonatal cause model states.
+   * - `Incidence`_
+     - Number of new cases of a given condition per person-year.
+     - Estimate transition rates.
+   * - `Remission`_
+     - Number of recovered cases from a given condition per person-year.
+     - Estimate transition rates.
+   * - `Disability Weights`_
+     - Proportion of full health not experienced due to disability associated
+       with a given condition.
+     - Measure disability attributed to cause model states.
+   * - `Cause-specific Mortality`_
+     -
+     -
+   * - `Excess Mortality`_
+     -
+     -
 
-Incidence
-+++++++++
+Prevalence
+++++++++++
+
+Prevalence is defined as the **proportion of a given population that possesses
+a given condition or trait** at a given time-point.
 
-Birth prevalence
+  For example, the prevalence of obesity in the United States was
```

Comment:
Use a cause as an example here.  When we talk about obesity and other risk factors, we call it exposure.